client/index.html

```diff
@@ -274,6 +274,14 @@ <h1 class="title">Settings</h1>
 									Enable notification for all messages
 								</label>
 							</div>
+
+							<div class="col-sm-12">
+								<label class="opt">
+									<label for="highlights" class="sr-only">Custom Highlights (separate by ",")</label>
```

Comment:
I suggest the following placeholder here and in the `<input />` below:

> Custom highlights (comma-separated keywords)

Reasoning is: no upper-casing of "highlight" to match other inputs/labels in this page, and using words instead of characters as 1) screen readers (at least VoiceOver) will not read them at all (it just reads "separated by") and the `','` looks weird, very packed on the default font.